plugins/node/opentelemetry-instrumentation-koa/README.md

```diff
@@ -41,6 +41,19 @@ registerInstrumentations({
 
 See [examples/koa](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/examples/koa) for a short example using both Koa and @koa/router
 
+### Koa Instrumentation Options
+
+Koa instrumentation has few options available to choose from. You can set the following:
+
+| Options | Type | Example | Description |
+| ------- | ---- | ------- | ----------- |
+| `ignoreLayersType`| `KoaLayerType[]` | `['middleware']` | Ignore layers of specified type. |
+
+`ignoreLayersType` accepts an array of following strings:
+
+- `router`,
+- `middleware`.
```

Comment:
`KoaLayerType` isn't really a useful type without explaining what it is. Either explain the type, or just say it is an array of string and list the possible values. Will koa users know what router and middleware layer types are or does that also need explanation?

Reply:
@dyladan thanks for the feedback. Updated the README to be more specific around what the values of `KoaLayerType` are in reference of.